bundles/org.openhab.binding.weathercompany/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0 http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:weathercompany:weather">
+		<parameter name="apiKey" type="text" required="true">
+			<label>API Key</label>
+			<description>Enter the API key</description>
+		</parameter>
+
+		<parameter name="locationType" type="text" required="true">
+			<label>Location Type</label>
+			<description>Location type (Postal Code, Geocode)</description>
+			<default>postal</default>
+			<options>
+				<option value="postalCode">Postal Code</option>
+				<option value="geocode">Geocode</option>
+				<option value="iataCode">IATA Code</option>
+			</options>
+		</parameter>
+
+		<parameter name="postalCode" type="text" required="false">
+			<label>Postal code</label>
+			<description>Postal code of form postal:country (e.g. 10001:US)</description>
+		</parameter>
+
+		<parameter name="geocode" type="text" required="false">
```

Comment:
Please define it just like here: https://github.com/openhab/openhab-core/blob/master/bundles/org.openhab.core/src/main/resources/ESH-INF/config/i18n.xml#L55-L64
This will make the UI to offer a map selection to the user.

Reply:
Fixed.

I should note that when saving the file, Eclipse wants to replace the tabs with spaces for indentation in the CDATA section here https://github.com/openhab/openhab-core/blob/master/bundles/org.openhab.core/src/main/resources/ESH-INF/config/i18n.xml#L59-L62. This causes SAT to fail the build. 🙄 I had to remove all indentation for the CDATA section to resolve the issue.